mautic/mautic
# Timeline event for queued messages

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds a new timeline event in the contact details to view queued messages and to be able to view them cancel them if the message is no longer relevant.

#### Steps to test this PR:
1. Apply this PR go to the contact's timeline and see the new message queue timeline event with details of when it was  first created, number of attempts, date that it will be rescheduled if it failed, and the ability to cancel them from the contact's timeline.
